doc/particles.md

```diff
@@ -214,46 +217,53 @@ game.add(
         particle: CircleParticle(
             radius: game.size.width / 2,
             paint: Paint()..color = Colors.red.withOpacity(.5),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Sprite Particle
+
 Allows you to embed Flame's `Sprite` into your particle effects. Useful when consuming graphics for the effect from `SpriteSheet`.
+
 ```dart
 game.add(
     ParticleComponent(
         particle: SpriteParticle(
           sprite: Sprite('sprite.png'),
           size: Vector2(64, 64),
-        )
-    )
+        ),
+    ),
 );
 ```
 
 ## Image Particle
+
 Renders given `dart:ui` image within the particle tree. 
+
 ```dart
 // During game initialisation
 await Flame.images.loadAll(const [
-    'image.png'
+    'image.png',
 ]);
 
 // ...
 
 // Somewhere during the game loop
+final image = await Flame.images.load('image.png');
+
 game.add(
     ParticleComponent(
         particle: ImageParticle(
-          size: const Size.square(24),
-          image: Flame.images.loadedFiles['image.png'],
+          size: Vector2.all(24),
+          image: image,
         );
-    )
+    ),
 );
 ```
 
 ## Animation Particle
+
 A `Particle` which embeds a Flame `Animation`. By default, aligns `Animation`s `stepTime` so that it's fully played during the `Particle` lifespan. It's possible to override this behavior with `alignAnimationTime` parameter.
```

Comment:
```suggestion
A `Particle` which embeds an `Animation`. By default, aligns the `Animation`'s `stepTime` so that it's fully played during the `Particle` lifespan. It's possible to override this behavior with the `alignAnimationTime` argument.
```